Formula/kibana.rb

```diff
@@ -28,18 +30,21 @@ def install
       system "make", "install"
     end
 
-    # do not build packages for other platforms
-    inreplace buildpath/"tasks/config/platforms.js", /('(linux-x64|windows-x64)',?(?!;))/, "// \\1"
+    # remove non open source files
+    rm_rf "x-pack"
 
     # trick the build into thinking we've already downloaded the Node.js binary
     mkdir_p buildpath/".node_binaries/#{resource("node").version}/darwin-x64"
 
-    # set npm env and fix cache edge case (https://github.com/Homebrew/brew/pull/37#issuecomment-208840366)
-    ENV.prepend_path "PATH", prefix/"libexec/node/bin"
-    system "npm", "install", "-ddd", "--build-from-source", "--#{Language::Node.npm_cache_config}"
-    system "npm", "run", "build", "--", "--release", "--skip-os-packages", "--skip-archives"
+    # run yarn against the bundled node version and not our node formula
+    (buildpath/"yarn").install resource("yarn")
+    ENV.prepend_path "PATH", buildpath/"yarn/bin"
+    system prefix/"libexec/node/bin/node", buildpath/"yarn/bin/yarn.js", "kbn", "bootstrap"
+    system prefix/"libexec/node/bin/node", buildpath/"yarn/bin/yarn.js", "build", "--oss",
+                                           "--release", "--skip-os-packages", "--skip-archives"
 
-    prefix.install Dir["build/kibana-#{version}-darwin-x86_64/{bin,config,node_modules,optimize,package.json,src,ui_framework,webpackShims}"]
+    prefix.install Dir["build/oss/kibana-#{version}-darwin-x86_64/{bin,config,node_modules,optimize,package.json,src,ui_framework,webpackShims}"]
+    prefix.install "licenses/APACHE-LICENSE-2.0.txt"
```

Comment:
Is there a way to prevent homebrew from installing the top level `LICENSE.txt` and overwriting the correct license, if I install the license here directly with: `prefix.install "licenses/APACHE-LICENSE-2.0.txt" => "LICENSE.txt"`?